filters/all.js

```diff
@@ -170,8 +171,10 @@ filter.convertToFilename = convertToFilename;
  * @param {Object} serverVariables object containing server variables.
  * @return {String}
  */
+/* eslint-disable */
```

Comment:
why do we need to disable here?

Reply:
we could actually remove this filter and import it from here https://github.com/asyncapi/generator-filters/blob/master/src/customFilters.js as one guy took it one day to main filters package. You could basically just remove this function and all should work as `generator-filters` are configured here already.

please just run generation locally and test it as described in readme, to be 100% sure all works like a charm